testFixtures/v3.1-RC/manifest.json

```
@@ -0,0 +1,156 @@
+{
+    "last_updated": "2019-07-04T13:33:03.969Z",
+    "ttl": 60,
+    "version": "3.1-RC",
+    "data": {
+        "datasets": [
+            {
+                "system_id": "tier_fingal",
+                "versions": [
+                    {
+                        "url": "https://platform-services.tier-services.io/data-sharing/tier_tier_fingal/gbfs/2.1",
```

I think it would be more neutral to use random names and URLs for the test files instead of those from a real operator. Thoughts @emmambd?

Example was found in https://github.com/MobilityData/gbfs/blob/master/gbfs.md#manifestjson.

```suggestion
                "system_id": "example_berlin",
                "versions": [
                    {
                        "url": "https://berlin.example.com/gbfs/2/gbfs",
```

---

@richfab Makes sense to me!

---

@Alessandro100 Can you please update the test files with examples from https://github.com/MobilityData/gbfs/blob/master/gbfs.md#manifestjson?

---

great idea